### PR TITLE
[zh-cn] sync kube-proxy.md

### PR DESCRIPTION
**Pull request by jiecloud**
sync content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md

Signed-off-by: Daniel Wan <jie.wan@daocloud.io>

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 1]
```diff
@@ -961,24 +961,11 @@ The previous version for which you want to show hidden metrics. Only the previou
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
-<!-- logs at or above this threshold go to stderr -->
-设置严重程度达到或超过此阈值的日志输出到标准错误输出。
+<!-- logs at or above this threshold go to stderr (no effect when -logtostderr=true or -alsologtostderr=false) -->
+设置严重程度达到或超过此阈值的日志输出到标准错误输出（当 -logtostderr=true 或 -alsologtostderr=false 时无效）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:965]:
```suggestion
设置严重程度达到或超过此阈值的日志输出到标准错误输出（当<code>--logtostderr=true</code>或
<code>--alsologtostderr=false</code>时无效）。
```

**Comment by tengqm** [comment]
Please squash your PR into a single commit.

**Comment by tengqm** [comment]
Please squash your PR into 1 commit. 14 commits is terrible.

**Comment by jiecloud** [comment]
sorry for this mess, I will close it  and open a new pr. 
thanks for the review @tengqm

**Pull request closed without merging**